src/components/User/index.js

```diff
@@ -0,0 +1,24 @@
+import React, { useState } from 'react'
+import GoogleLogin from 'react-google-login'
+
+const User = () => {
+    const [isLoggedin, setIsLoggedin] = useState(false)
+
+    const buttonText = !isLoggedin ? 'Login' : 'Logout';
+
+    const handleResponse = (reponse) => {
+        console.log(reponse)
+    }
+
+    return (
+        <GoogleLogin 
+            clientId="612923311848-gb7hs3mac93dqtecdpn0qq6fhnbvrl8d.apps.googleusercontent.com"
```

Comment:
Is there a way to secure this clientId? Client id isn't necessarily the biggest deal, but we should move any important keys to a separate file and set those through CI when the time comes.